src/main/java/teammates/common/util/Const.java

```diff
@@ -1010,7 +1010,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_RESPONSE_INVALID_RECIPIENT = "Trying to update recipient to an invalid recipient for question %d.";
         public static final String FEEDBACK_RESPONSE_RECIPIENT_ALREADY_EXISTS = "Error trying to update recipient for response, as another response with the same recipient already exists.";
         
-        public static final String FEEDBACK_SUBMISSIONS_NOT_OPEN = "You can view the questions and any submitted responses for this feedback session but cannot submit new responses as the session is not currently open for submission.";
+        public static final String FEEDBACK_SUBMISSIONS_NOT_OPEN = "<strong>The feedback session is currently closed.</strong> You can view the questions and any submitted responses for this feedback session but cannot submit new responses.";
```

Comment:
'not open for submission' is better? Sometimes sessions are opened by accident and then put back into awaiting state (which is technically not the same as 'closed')